### ISSUE TITLE
Cannot find fallback server / "no fallback server"

### ISSUE THREAD

**Issue by nooneyy** [AoElite/AutoReconnect#20]
**Hi,
Whenever I restart my spigot server, the plugin fails to connect me to my fallback server.
I'm using the newest version of AutoReconnect - 1.6.3, Paper 1.16.3/#253 and Waterfall #380 along with Protocolize in my Paper plugins folder.**

Here's my Waterfall/Bungee config:
listeners:
- query_port: 25567
  motd: '&1mincecraft server :)'
  tab_list: GLOBAL_PING
  query_enabled: false
  proxy_protocol: false
  forced_hosts:
    pvp.md-5.net: pvp
  ping_passthrough: false
  priorities:
  - survival
  - hub
  bind_local_address: true
  host: 0.0.0.0:25565
  max_players: 69
  tab_size: 60
  force_default_server: false
remote_ping_cache: -1
network_compression_threshold: 256
permissions:
  default:
  - bungeecord.command.server
  - bungeecord.command.list
  admin:
  - bungeecord.command.alert
  - bungeecord.command.end
  - bungeecord.command.ip
  - bungeecord.command.reload
log_pings: true
connection_throttle_limit: 3
server_connect_timeout: 5000
timeout: 30000
stats: e904d57f-ba11-4979-9159-6bfea4d09c91
player_limit: -1
ip_forward: true
groups:
  md_5:
  - admin
remote_ping_timeout: 5000
connection_throttle: 4000
log_commands: false
prevent_proxy_connections: false
online_mode: false
forge_support: true
disabled_commands:
- disabledcommandhere
servers:
  survival:
    motd: '&1mincecraft server :)'
    address: localhost:25566
    restricted: false
  hub:
    motd: '&1minecraft server :)'
    address: localhost:25569
    restricted: false

And here's my Waterfall/Bungee log (only the important stuff, autoreconnect debug enabled):

[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Attempting to reconnect user nooney_  to survival because of kick: Server closed
[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Reconnecting user nooney_ to survival because of kick: Server closed
[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Attempting to move to limbo
[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Version 1.16 or greater
[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Version 1.16.2 or greater
[23:32:23] [Netty Worker IO Thread #2/INFO] [AutoReconnect]: Keeping user connection alive nooney_ 

[23:32:54] [AutoReconnect Pool Thread #2/INFO] [AutoReconnect]: Reconnect attempt 17 for nooney_ to survival
[23:32:54] [AutoReconnect Pool Thread #2/INFO] [AutoReconnect]: Max tries reached for nooney_ to survival
[23:32:54] [AutoReconnect Pool Thread #2/INFO] [AutoReconnect]: Cancelling reconnect task for <insert uuid here >
[23:32:54] [AutoReconnect Pool Thread #2/INFO] [AutoReconnect]: No fallback server for nooney_

And here's what happens to me in game:
![image](https://user-images.githubusercontent.com/26509743/97818713-430be780-1ca4-11eb-9b83-56b1048aba7a.png)

**Thanks in advance.**

**Comment by PseudoResonance** [comment]
Yes, this is currently not working. I am planning to rewrite much of this plugin sometime soon so that it is more efficient and works better.

At the moment, instead of being properly sent to the fallback server, you end up stuck in the limbo server, but you can always switch servers with /server.
